packages/core/docs/guide/theme.md

```diff
@@ -94,3 +94,41 @@ Below you can find a list of the most important Nuxt Modules and libraries that
 
 - [`@storefront-ui/vue`](https://storefrontui.io)
 - [`wee-validate`](https://vee-validate.logaretm.com/v3)
+
+## In-app Notifications
+
+In Vue Storefront we're providing Notifications system based on composable `useUiNotification`.
+
+There are three types of notifications: `danger`, `success` and `info`.
+
+### How to send notification?
```

Comment:
missing info about "common notifications" in assets/notifications with sendNotification

Reply:
Added :)